doc/material-controls-extensions.md

```diff
@@ -36,4 +43,91 @@ It's control specific and for now, you can only use it with the ToggleButton con
             <PathIcon Data="{StaticResource Icon_more_vertical}" />
         </um:ControlExtensions.AlternateContent>
     </ToggleButton>
-```
\ No newline at end of file
+```
+
+## Elevation
+
+This feature allows for setting the level of elevation to depict on the supported control.
+
+Setting the elevation on [supported controls]() can result in changes to both the shadow and the [surface tint](#surface-tint).
+
+[Material Design Elevation Guidance](https://m3.material.io/styles/elevation/overview)
+
+## Surface Tint
+
+The Surface Tint properties allow for customization of how elevation can be depicted on certain controls. While the Background of a control remains static, the surface color can change based on the level of elevation.
+
+### TintedBackground
+
+This is a readonly property that will provide a SolidColorBrush depicting the control's current background color overlayed with the Surface Tint color at a certain opacity based on the elevation of the control.
+
+### IsTintEnabled
+
+This feature allows for enabling or disabling the surface tint that may be applied to an elevated control. When `IsTintEnabled` is `false`, the `TintedBackground` property will remain the same value as the control's background color.

```

Comment:
In `## Surface Tint`, you were using "Surface Tint" and "Background", here they are normal case. 
TitleCase or normal case, you got to pick a side.